nototools/cldr_data.py

```diff
@@ -509,14 +512,18 @@ def get_exemplar_from_file(cldr_file_path):
     _exemplar_from_file_cache[cldr_file_path] = None
     return None
 
-  exemplars = None
+  exemplars = set()
   for tag in root.iter('exemplarCharacters'):
     if 'type' in tag.attrib:
+      typeval = tag.attrib['type']
+    else:
+      typeval = ''
```

Comment:
This could now be `typeval = tag.attrib.get('type', '')` if tag is a normal dictionary.